sdk/lib/rtl/actctx.c

```diff
@@ -9,7 +9,7 @@
  *                  Jacek Caban for CodeWeavers
  *                  Alexandre Julliard
  *                  Stefan Ginsberg (stefan.ginsberg@reactos.org)
- *                  Samuel Serapi�n
+ *                  Samuel Serapión
```

Comment:
Unintended change?

Reply:
Sort of. My editor replaced the `ó` [with a replacement character](https://github.com/reactos/reactos/blob/457165eaad137e6a6aa0e7149c5d49dead19b441/sdk/lib/rtl/actctx.c#L12) for some reason, so I replaced the original letter in d68cbf6. I didn't see it in the diff when I prepared the PR for some reason.